apps/src/componentLibrary/dropdown/simpleDropdown/SimpleDropdown.tsx

```diff
@@ -3,6 +3,7 @@ import React, {AriaAttributes} from 'react';
 
 import {getAriaPropsFromProps} from '@cdo/apps/componentLibrary/common/helpers';
 import {ComponentSizeXSToL} from '@cdo/apps/componentLibrary/common/types';
+import Spinner from '@cdo/apps/sharedComponents/Spinner';
```

Comment:
When I've done small refactors/updates of our design system components like this, Denys (@levadadenys ) has requested that I follow a specific procedure. Would it be alright if we pulled this part out into its own PR and tagged him in it? I believe the ideal PR would include the change, an update to the corresponding changelog, and updates of any storybook components to show how your updates work!